### ISSUE TITLE
Loaded world disappears after a few seconds on windows

### ISSUE THREAD

**Issue by OperationDarkside** [Hopson97/MineCraft-One-Week-Challenge#30]
First of all thank you for your work.

Second, I cloned this repository and tried to get it to work on windows with visual studio 2015. I didn't use cmake because it kept throwing errors I didn't fully understand. I downloaded glew, glm and sfml and compiled those and included them into my project. After compiling the project and tweaking the variables a little and adding dll's I got it running in debug mode. Well, at least showing a window. The window shows a blue background, after a few seconds the generated world appears and 3 seconds later it's back to the world background (blue sky) and the fps are awefull. I imagine, I made a mistake while linking.

Now the actual question: Is that a problem for you as well? If not, do you have a .sln file, that could help me?

**Comment by Hopson97** [comment]
I have never heard of that happening before, so I am not sure. (I am on Windows, works fine for me). I do not have a .sln file, as I don't use VS

**Comment by Hopson97** [comment]
+ I don't use Cmake either :S

**Comment by IridescentRose** [comment]
@OperationDarkside I'll try it out with windows and tell you what I did today.

**Comment by OperationDarkside** [comment]
You are my hero today. I don't know ow to use make on windows.

**Comment by IridescentRose** [comment]
:) I'll be doing stuff on windows after I finish the graphics mod.

**Comment by NT-Bourgeois-Iridescence-Technologies** [comment]
Steps So Far:

1. Ran `git clone` using [Git For Windows](https://git-scm.com/download/win) in Command Prompt
2. Installed [MinGW-w64](https://sourceforge.net/projects/mingw-w64/) with the install settings: Version 6.1.0, Architecture x86_64, Threads POSIX, Exception SEH, and build revision 1.
3. Installed GCC to command prompt by adding `C:/path to mingw/bin` to my Windows path.
4. Downloaded the same version of [SFML](https://www.sfml-dev.org/files/SFML-2.4.2-windows-gcc-6.1.0-mingw-64-bit.zip) to match the compiler (lower right corner).

**Comment by IridescentRose** [comment]
@OperationDarkside  Okay, one quick thing. Make sure render distance is down to 6-8 because 16 causes instability issues with my system (which is fairly beefy). You can modify this in the config.txt file. There is a bug which causes you to trip out of the world under low framerates...

**Comment by NT-Bourgeois-Iridescence-Technologies** [comment]
Sometimes that causes you to instantly flip out of the world because... logic? And basically crashes the app from there.

**Comment by IridescentRose** [comment]
I can confirm it working with Visual Studio. A little glitchy with looking around, but I imagine using DevC++ or Code::Blocks would work better, as @Hopson97 used Code::Blocks

![capture](https://user-images.githubusercontent.com/12397073/30516621-f8a9b710-9b11-11e7-8d83-09128dcba5a3.PNG)

**Comment by OperationDarkside** [comment]
Hello,

sorry for the long pause. I tested a lot with visual studio and didn't get a satisfactory result. The reduced render distance helped quite much, but the world kept disappearing too often.

So I tried Code::Blocks and after some time with configuring MingW I got it to compile. Strangely the world/blocks didn't disappear (still had to link dynamically). But the problem was i think mainly Nr. 40. 40 cm in front of the screen and my wish to use vs.

So this issue is kind of closed. Thank you for any help
